### PR TITLE
Fix1256 : Update the link to 'getstarted' page

### LINKED ISSUE #1256: Side navigation bar is missing in certain versions of documentation
Side navigation bar is missing in the following versions of documentation
1.0.0, 0.9.0, 0.8.0 and 0.7.0

It works ok with rest of versions.

Also the landing page on these above versions is 'overview' whereas it should have been 'getstarted'

Will be sending a PR to fix it, later on.

### PR DESCRIPTION
**Pull request by k8s-dev**
This fixes : https://github.com/litmuschaos/litmus/issues/1256

### CODE REVIEW

File: website/versioned_docs/version-1.0.0/openebs-pool-pod-failure.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
+---
```
Comment (ksatchit) [inline website/versioned_docs/version-1.0.0/openebs-pool-pod-failure.md]: The 1.0.0 version of the chaosengine doesn't support this schema described here. Looks like the 1.1 version

File: website/versioned_docs/version-1.0.0/openebs-pool-pod-failure.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+  </tr>
+  <tr>
+    <td> DATA_PERSISTENCE </td>
+    <td> Flag to perform data consistency checks on the application </td>
+    <td> Optional  </td>
+    <td> Default value is disabled (empty/unset). It supports only `mysql` and `busybox`. Ensure configmap with app details are created </td>
+  </tr>
+</table>
+
+#### Sample ChaosEngine Manifest
```
Comment (ksatchit) [inline website/versioned_docs/version-1.0.0/openebs-pool-pod-failure.md]: The 1.0.0 version of the chaosengine doesn't support this schema described here. Looks like the 1.1 version

File: website/versioned_docs/version-1.0.0/openebs-target-network-delay.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+  </tr>
+  <tr>
+    <td> DATA_PERSISTENCE </td>
+    <td> Flag to perform data consistency checks on the application </td>
+    <td> Optional  </td>
+    <td> Default value is disabled (empty/unset). It supports only `mysql` and `busybox`. Ensure configmap with app details are created </td>
+  </tr>
+</table>
+
+#### Sample ChaosEngine Manifest
```
Comment (ksatchit) [inline website/versioned_docs/version-1.0.0/openebs-target-network-delay.md]: The 1.0.0 version of the chaosengine doesn't support this schema described here. Looks like the 1.1 version

File: website/versioned_docs/version-1.0.0/openebs-target-network-loss.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+  </tr>
+  <tr>
+    <td> DATA_PERSISTENCE </td>
+    <td> Flag to perform data consistency checks on the application </td>
+    <td> Optional  </td>
+    <td> Default value is disabled (empty/unset). It supports only `mysql` and `busybox`. Ensure configmap with app details are created </td>
+  </tr>
+</table>
+
+#### Sample ChaosEngine Manifest
```
Comment (ksatchit) [inline website/versioned_docs/version-1.0.0/openebs-target-network-loss.md]: The 1.0.0 version of the chaosengine doesn't support this schema described here. Looks like the 1.1 version

File: website/versioned_docs/version-1.0.0/openebs-target-pod-failure.md [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+  </tr>
+  <tr>
+    <td> DATA_PERSISTENCE </td>
+    <td> Flag to perform data consistency checks on the application </td>
+    <td> Optional  </td>
+    <td> Default value is disabled (empty/unset). It supports only `mysql` and `busybox`. Ensure configmap with app details are created </td>
+  </tr>
+</table>
+
+#### Sample ChaosEngine Manifest
```
Comment (ksatchit) [inline website/versioned_docs/version-1.0.0/openebs-target-pod-failure.md]: The 1.0.0 version of the chaosengine doesn't support this schema described here. Looks like the 1.1 version

**Comment by ksatchit** [comment]
Hi @k8s-dev - thank you for this PR. Looks good for most part, the OpenEBS 1.0 versioned docs seems to contain the latest chaosengine spec. They are appropriate as defined here: https://docs.litmuschaos.io/docs/1.0.0/openebs-target-container-failure/

**Comment by k8s-dev** [comment]
Updated with the required changes to reflect v1.0.0 of ChaosEngine format.

File: website/versioned_docs/version-1.0.0/openebs-pool-pod-failure.md [hunk 6]
```diff
@@ -1,8 +1,8 @@
 ---
-id: version-0.9.0-openebs-pool-container-kill
-title: OpenEBS Pool Container Kill Experiment Details
-sidebar_label: Pool Container Kill
-original_id: openebs-pool-container-kill
+id: version-1.0.0-openebs-pool-pod-failure
```
Comment (chandankumar4) [inline website/versioned_docs/version-1.0.0/openebs-pool-pod-failure.md:2]: Thanks, @k8s-dev for the refactoring PR.
Can you update this file with the `1.1.0-openebs-pool-pod-failure` as it contains the `openebs-pool-container-failure` name also in line no: 18

**Review by chandankumar4 — APPROVED** [review]
/approve
/lgtm
Thanks @k8s-dev

**Pull request merged**